### PR TITLE
Feature/enable node auto repair

### PR DESCRIPTION
**Pull request by kushZemoso**
# Description

Ensure that Auto-Repair feature is enabled for all your Google Kubernetes Engine (GKE) cluster nodes in order to help you keep the cluster nodes healthy. Google Kubernetes Engine uses the node's health status to determine if a cluster node needs to be repaired. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] The commit message/PR follows our guidelines

# **Other information**:

### CODE REVIEW

File: installer/resources/pacbot_app/files/DB.sql [hunk 1]
```diff
@@ -1708,7 +1708,8 @@ INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,
...
-INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('Set_all_users_to_owner_email_notifications','Set All users with the following roles to owner in email notifications for MS defender.','In MS defender email alert settings, Ensure All users with following rules is set to owner.',NULL,NULL,'version-1',NULL,NULL,'2022-10-14','2022-10-14','https://github.com/PaladinCloud/CE/wiki/Azure-Policy#in-ms-defender-ensure-that-all-users-with-the-following-roles-is-set-to-owner');
```
Comment (dheerajkholia-paladin) [inline installer/resources/pacbot_app/files/DB.sql:1711]: Please verify if this deletion is needed

File: jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/GKEClusterRule/EnableNodeAutoUpgrade.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            }
+
+        }
+
+        return validationResult;
+    }
+    @Override
+    public String getHelpText() {
+        return "Enable  GKE nodes management properties";
+    }
```
Comment (dheerajkholia-paladin) [inline jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/GKEClusterRule/EnableNodeAutoUpgrade.java:119]: Some of the changes are already added with #609, please rebase once to master

File: installer/resources/pacbot_app/files/DB.sql (hunk 1 above)
Comment (kushZemoso) [reply installer/resources/pacbot_app/files/DB.sql:1711]: yes we have a duplicated record  for this policy.

File: jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/GKEClusterRule/EnableNodeAutoUpgrade.java (hunk 2 above)
Comment (kushZemoso) [reply jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/GKEClusterRule/EnableNodeAutoUpgrade.java]: Done

**Review by dheerajkholia-paladin — APPROVED** [review]

**Review by santhosh-challa — APPROVED** [review]

**Pull request merged**